docs/usage.md

```diff
@@ -240,6 +247,7 @@ Similar to number formatting, you can format plain dates and times that are not
 
 ```js
 import {useIntl} from 'next-intl';
+import {parseISO} from 'data-fns';
```

Comment:
Do you mean `date-fns` ?

Reply:
Yep 🤦. Fixed now!

Btw. I'm currently migrating the docs to a standalone website. There are no major changes yet, but I think it's already easier to browse in comparison to the markdown files. So if you're interested in an early preview: https://next-intl-amann.vercel.app/.

Reply:
YES! and I think you'll be able to include interactive snippets there right? via codesandbox or something. So it will be even better for users being able to read and play and see it in actions. I'm using radix-ui on my current project and their documentation website is really good if you're after a reference. https://www.radix-ui.com/ (this is their "sales pitch" site, but then you can jump straight into the documentation. Here's an example of their documentation with interactive examples https://www.radix-ui.com/docs/primitives/components/dialog

Its really nice to be able to test something without having to setup a project, install dependencies etc.

Reply:
Haha, sounds promising 😁. Yes, I definitely remember when you wrote me on Twitter! I'll see if I can get the docs shaped up a bit and I'll let you know when I've got something to show! Thank you for your great inputs! 🙂